cli/shared/src/test/scala/ch/epfl/bluebrain/nexus/cli/SparqlClientSpec.scala

```diff
@@ -1,6 +1,85 @@
 package ch.epfl.bluebrain.nexus.cli
 
+import cats.effect.IO
+import ch.epfl.bluebrain.nexus.cli.ClientError.{ClientStatusError, ServerStatusError}
+import ch.epfl.bluebrain.nexus.cli.SparqlClient._
+import ch.epfl.bluebrain.nexus.cli.types.{Label, SparqlResults}
+import ch.epfl.bluebrain.nexus.cli.utils.Fixtures
+import org.http4s.Method._
+import org.http4s.circe.CirceEntityEncoder._
+import org.http4s.client.Client
+import org.http4s.headers._
+import org.http4s.{HttpApp, Response, Status}
+import org.scalatest.OptionValues
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpecLike
 
-class SparqlClientSpec extends AnyWordSpecLike with Matchers {}
+class SparqlClientSpec extends AnyWordSpecLike with Matchers with Fixtures with OptionValues {
+
+  private val organization  = Label("myorg")
+  private val project       = Label("mylabel")
+  private val query         = "SELECT * {?s ?p ?o} LIMIT 10"
+  private val randomViewUri = nxv / genString()
+
+  private val mockedHttpApp = HttpApp[IO] {
```

Comment:
maybe using `mockito` would be a bit cleaner here?

Reply:
Yes, maybe. I tried to do it without mockito from what we discussed lately. But this case is simple enough to be able to do it in both ways.

Let's see what @bogdanromanx thinks.

Reply:
I very much prefer this implementation over a mockito impl.